dasharo-security/secure-boot.robot

```diff
@@ -129,281 +121,210 @@ SBO003.001 Attempt to boot file with the correct key from Shell (firmware)
+    ${key_menu}=    Enter Key Management And Return Construction
+    Enroll DB Signature    ${key_menu}    GOOD_KEYS    cert.der
+    Save Changes And Reset
+    Boot Efi File    signed-hello.efi    GOOD_KEYS    Hello, World!
 
 SBO004.001 Attempt to boot file without the key from Shell (firmware)
     [Documentation]    This test verifies that Secure Boot blocks booting a file
     ...    without a key.
     Skip If    not ${SECURE_BOOT_SUPPORT}    SBO004.001 not supported
     Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO004.001 not supported
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SBO004.001 not supported
-    Mount ISO As USB In QEMU    ${CURDIR}/../scripts/secure-boot/images/NOT_SIGNED.img
+    Mount ISO As USB    ${CURDIR}/../scripts/secure-boot/images/NOT_SIGNED.img
     Power On
-    Enter UEFI Shell
-    ${out}=    Execute File In UEFI Shell    hello.efi
-    Should Contain    ${out}    Access Denied
+    Boot Efi File Should Fail    hello.efi    NOT_SIGNED
 
 SBO005.001 Attempt to boot file with the wrong-signed key from Shell (firmware)
     [Documentation]    This test verifies that Secure Boot disallows booting
     ...    a signed file with a wrong-signed key.
     Skip If    not ${SECURE_BOOT_SUPPORT}    SBO005.001 not supported
     Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO005.001 not supported
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SBO005.001 not supported
-    Mount ISO As USB In QEMU    ${CURDIR}/../scripts/secure-boot/images/BAD_KEYS.img
+    Mount ISO As USB    ${CURDIR}/../scripts/secure-boot/images/BAD_KEYS.img
     Power On
-    Enter UEFI Shell
-    ${out}=    Execute File In UEFI Shell    signed-hello.efi
-    Should Contain    ${out}    Access Denied
+    Boot Efi File Should Fail    signed-hello.efi    BAD_KEYS
 
 SBO006.001 Reset Secure Boot Keys option availability (firmware)
     [Documentation]    This test verifies that the Reset Secure Boot Keys
     ...    option is available
     Skip If    not ${SECURE_BOOT_SUPPORT}    SBO006.001 not supported
     Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO006.001 not supported
-    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SBO006.001 not supported
     Power On
-    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
-    ${device_mgr_menu}=    Enter Submenu From Snapshot And Return Construction
-    ...    ${setup_menu}
-    ...    Device Manager
-    ${sb_menu}=    Enter Submenu From Snapshot And Return Construction
-    ...    ${device_mgr_menu}
-    ...    Secure Boot Configuration
-    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
-    Should Contain    ${advanced_menu}    > Reset to default Secure Boot Keys
+    Enter Secure Boot Menu
+    ${key_menu}=    Enter Key Management And Return Construction
+    Should Contain    ${key_menu}    ${RESET_KEYS_OPTION}
 
 SBO007.001 Attempt to boot the file after restoring keys to default (firmware)
```

Comment:
@m-iwanicki Faced following error while running tests on commit `236b638a06073132378566a98d62c37195c7c83e`:

```bash
(...)
SBO007.001 Attempt to boot the file after restoring keys to defaul... | FAIL |
No match found for 'Esc=Exit' in 30 seconds. Output:
                   |
| LCtrl+LAlt+F12=Save screenshot                                               |
\------------------------------------------------------------------------------/^v=Move Highlight



                                                         Current Secure Boot
                                                         state: enabled or
                                                         disabled.
(...)
```

Reply:
@TomaszAIR I might not be able to check if this commit has resolved the issue due to configuration problems in QEMU I am testing this on. So test it please and resolve this thread if it fixes the issue.

Reply:
That happened because of this issue #154, fixed in [754c180](https://github.com/Dasharo/open-source-firmware-validation/pull/194/commits/754c180a820e4e8826bd2d004c96000e5c256829).

Reply:
The test passed:

```shell
==============================================================================
Secure-Boot                                                                   
==============================================================================
SBO007.001 Attempt to boot the file after restoring keys to defaul... | PASS |
------------------------------------------------------------------------------
Secure-Boot                                                           | PASS |
1 test, 1 passed, 0 failed
==============================================================================
```

Resolving this thread.